image/cli/mascli/templates/gitops/cis-compliance.yaml.j2

```diff
@@ -0,0 +1,27 @@
+---
+# IBM Cloud Pak Foundational Services
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: cis-compliance
+  namespace: openshift-gitops
+  finalizers:
+    - resources-finalizer.argocd.argoproj.io
+spec:
+  project: {{ CLUSTER_ID }}-cluster-scope
+  destination:
+    server: https://kubernetes.default.svc
+    namespace: cis-compliance
+  source:
+    repoURL: https://github.com/ibm-mas/gitops
+    path: applications/cis-compliance
+    targetRevision: {{ GITOPS_VERSION }}
+  syncPolicy:
+    automated:
+      prune: true
+      selfHeal: true
+    syncOptions:
+      - CreateNamespace=false
+    managedNamespaceMetadata:
```

Comment:
This `managedNamespaceMetadata` section we need to remove as it was causing issues. That change has come in recently after you did these changes. Just delete lines 25 to 27